### PR TITLE
Added use case for storage limit

### PR DESCRIPTION
**Pull request by hussnain612**
Added use case to restrict storage limits and storage classes for tenants

### CODE REVIEW

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-74-a31a06e0`

File: docs/content/sre/tenant-operator/usecases/volume-limits.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  quota: medium
+  sandbox: true
+EOF
+```
+
+Now, the combined storage used by all tenant namespaces will not exceed `50Gi`. 
+
+### Adding StorageClass Restrictions for Tenant
+
+Now, Bill, as a cluster admin, wants to make sure that no Tenant can provision more than a fixed amount of storage from a StorageClass. Bill can restrict that using `<storage-class-name>.storageclass.storage.k8s.io/requests.storage` field in `quota.spec.resourcequota.hard` field. If Bill wants to restrict tenant `sigma` to use only `20Gi` of storage from storage class `stakater`, he'll first create a StorageClass `stakater` and then create the relevant Quota with `sigma.storageclass.storage.k8s.io/requests.storage` field set to `20Gi`.
```
Comment (MuneebAijaz) [inline docs/content/sre/tenant-operator/usecases/volume-limits.md]:
```suggestion
Now, Bill, as a cluster admin, wants to make sure that no Tenant can provision more than a fixed amount of storage from a StorageClass. Bill can restrict that using `<storage-class-name>.storageclass.storage.k8s.io/requests.storage` field in `quota.spec.resourcequota.hard` field. If Bill wants to restrict tenant `sigma` to use only `20Gi` of storage from storage class `stakater`, he'll first create a StorageClass `stakater` and then create the relevant Quota with `stakater.storageclass.storage.k8s.io/requests.storage` field set to `20Gi`.
```

File: docs/content/sre/tenant-operator/usecases/volume-limits.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+apiVersion: tenantoperator.stakater.com/v1beta1
+kind: Quota
+metadata:
+  name: small
+spec:
+  resourcequota:
+    hard:
+      requests.cpu: '2'
+      requests.memory: '4Gi'
+      sigma.storageclass.storage.k8s.io/requests.storage: '20Gi'
```
Comment (MuneebAijaz) [inline docs/content/sre/tenant-operator/usecases/volume-limits.md]:
```suggestion
      stakater.storageclass.storage.k8s.io/requests.storage: '20Gi'
```

**Review by MuneebAijaz — CHANGES_REQUESTED** [review]

**Review by MuneebAijaz — APPROVED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-74-2d43a7ac`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.48`

**Pull request merged**